README.rst

```diff
@@ -176,7 +176,8 @@ command::
 
 To run the application, use the ``paste`` script with the ``serve`` command.
 (The paste script and serve command come from ``PasteScript`` and
-``PasteDeploy``, respectively.)
+``PasteDeploy``, respectively, which will need to be installed: ``pip install pastescript``
+``pip install pastedeploy``.)
```

Comment:
I don't know much about `pserve`, so feel free to suggest a revision that is clearer. If it makes sense to reject this particular change, that's fine.